cis_v120/section_4.sp

```diff
@@ -0,0 +1,158 @@
+locals {
+  cis_v120_4_common_tags = merge(local.cis_v120_common_tags, {
+    cis_section_id = "4"
+  })
+}
+
+locals {
+  cis_v120_4_1_common_tags = merge(local.cis_v120_4_common_tags, {
+    cis_section_id = "4.1"
+  })
+  cis_v120_4_2_common_tags = merge(local.cis_v120_4_common_tags, {
+    cis_section_id = "4.2"
+  })
+  cis_v120_4_3_common_tags = merge(local.cis_v120_4_common_tags, {
+    cis_section_id = "4.3"
+  })
+}
+
+benchmark "cis_v120_4" {
+  title         = "4 Object Storage"
+  documentation = file("./cis_v120/docs/cis_v120_4.md")
+  children = [
+    benchmark.cis_v120_4_1,
+    benchmark.cis_v120_4_2,
+    benchmark.cis_v120_4_3,
+  ]
+
+  tags = merge(local.cis_v120_4_common_tags, {
+    service = "OCI/Storage"
+    type    = "Benchmark"
+  })
+}
+
+benchmark "cis_v120_4_1" {
+  title         = "4.1 Object Storage"
```

Comment:
Is the title correct? I see a similar title being used for `cis_v120_4` benchmark.